src/coreclr/src/dlls/mscorrc/mscorrc.rc

```diff
@@ -500,6 +475,7 @@ BEGIN
     IDS_EE_MISSING_METHOD                   "Method not found: '%1'."
 
     IDS_EE_NO_IDISPATCH                     "This object cannot be marshaled as an IDispatch."
+    IDS_EE_NO_IINSPECTABLE                  "Marshalling as IInspectable is not supported in the .NET runtime."
```

Comment:
>IDS_EE_NO_IINSPECTABLE [](start = 4, length = 22)

Haven't gotten this far yet, but note we should write a test for this.

Reply:
I've added a few tests.